modeshape-jcr/src/main/java/org/modeshape/jcr/mimetype/tika/TikaMimeTypeDetector.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.modeshape.jcr.mimetype;
+package org.modeshape.jcr.mimetype.tika;
```

Comment:
Are you sure there are no backward compatibility issues or required configuration changes with this moving of the classes?